.github/workflows/ci.yml

```diff
@@ -189,7 +189,7 @@ jobs:
           - os: windows
             runner: windows-latest
           - os: linux
-            runner: n1-standard-8-netssd-preempt
+            runner: [ self-hosted, linux, amd64, "4" ]
```

Comment:
you may just define it like this, any core count runner should be used then going forward
```suggestion
            runner: [ self-hosted, linux, amd64 ]
```

Reply:
Thanks :+1: